cortex/cli.py

```
@@ -261,6 +267,306 @@ def rollback(self, install_id: str, dry_run: bool = False):
+            if not template:
+                self._print_error(f"Template '{template_name}' not found")
+                self._print_status("[*]", "Available templates:")
+                templates = template_manager.list_templates()
+                for name, info in templates.items():
+                    print(f"  - {name}: {info['description']}")
+                return 1
+            
+            # Display template info
+            print(f"\n{template.name} Template:")
+            print(f"   {template.description}")
+            print(f"\n   Packages:")
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)